pychromecast/controllers/media.py

```diff
@@ -67,6 +69,15 @@ def __init__(self):
         self.media_metadata = {}
         self.subtitle_tracks = {}
         self.current_subtitle_tracks = []
+        self.last_updated = time.time()
```

Comment:
I think we should have `last_updated` be invalid (i.e. `None`) when no information has been received yet to indicate **unknown state**.